src/binary/logger.rs

```diff
@@ -108,7 +108,10 @@ impl Logger {
                 if self.y_pos >= (self.height() - BITMAP_LETTER_WIDTH) {
                     self.clear();
                 }
-                let bitmap_char = get_bitmap(c, FontWeight::Regular, BitmapHeight::Size14).unwrap();
+                let bitmap_char = get_bitmap(c, FontWeight::Regular, BitmapHeight::Size14)
+                    .unwrap_or(
+                        get_bitmap('■', FontWeight::Regular, BitmapHeight::Size14).unwrap()
```

Comment:
Maybe � (Replacement Character)?

Reply:
What are you referring? � it's not in the noto-sans-mono-bitmap crate yet.

Reply:
> it's not in the noto-sans-mono-bitmap crate yet.

Really? I would have expected it to be. Guess it is fine to leave it like this.

Reply:
OK, later we can ask to @phip1611 for adding that char. Two days ago the other char was not.
See this [issue](https://github.com/phip1611/noto-sans-mono-bitmap-rs/issues/4) for more information.

Reply:
> Really? I would have expected it to be. Guess it is fine to leave it like this.

Maintainer here: I'm about to add unicode ranges as selectable cargo features. Thus, it will be available soon, hopefully.

Reply:
tho... What?

Reply:
> tho... What?

https://www.urbandictionary.com/define.php?term=tho :)

Reply:
Sorry, I never listened that. I'm from Spain and English in Spain is quite bad.
Thanks, now I have one word more.

Reply:
So we merge it, or we wait for the other char which is more descriptive?

Reply:
Update regarding unicode features in the `noto-sans-mono-bitmap`. I experimented with it but even adding a "reasonable amount" of unicode ranges results in a 12 GB large crate, as all the characters must be pre-rasterized and shipped with the crate. 

This is unacceptable. I decided that `noto-sans-mono-bitmap` has to strive for minimality. If you want the whole unicode range @asensio-project , switch to user-mode and use the floating point unit. You will be able to use dynamic rendering of all the chars you can think of.

I'll drop the whole approach but update the crate documentation

Reply:
OK, I will create a PR for adding the � char to noto-sans-mono-bitmap-rs and then we can merge it for the main branch and the next branch of the bootloader. @phip1611